src/js/components/FormField/FormField.js

```diff
@@ -494,6 +494,9 @@ const FormField = forwardRef(
         {...outerProps}
         style={outerStyle}
         onFocus={(event) => {
+          if (!formFieldRef.current) {
```

Comment:
I'm thinking that even if `formFieldRef.current` is undefined we should still run line 504 `if (onFocus) onFocus(event);`.

Maybe we could change this section to be:
``` javascript
onFocus={(event) => {
  const root = formFieldRef.current?.getRootNode();
  if (root) {
    setFocus(
      containsFocus(formFieldRef.current) && shouldKeepFocus(root),
    );
  }
  if (onFocus) onFocus(event);
}}
```

Reply:
ahh yeah looks safer, thanks